webapp/src/components/ContactSupportMenu.js

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from "react";
+import { Form } from "react-bootstrap";
+import { useForm } from "react-hook-form";
+
+import "react-widgets/styles.css";
+import EnvironmentVars from "../EnvironmentVars";
+
+import "./css/ContactSupportMenu.css"
+
+const ContactSupportMenu = () => {
+    const [hidden, setHidden] = useState(true); // hidden by default
+    const [successMsg, setSuccessMsg] = useState("");
+    const [errorState, setErrorState] = useState(false);
+    const [errorMessage, setErrorMessage] = useState("");
+    const {
+        register,
+        handleSubmit,
+        reset,
+        formState: { errors },
+    } = useForm();
+
+    const sendPostData = async (json) => {

```

Comment:
This whole JS file is handling state management within itself and only itself. This doesn't follow our new redux design that we are utilizing everywhere else. This is a feature that doesn't pair with any other component in the site so it is very isolated. Due to this, I think it is probably okay to handle state management within the JS file itself. I would recommend breaking this function out into our API calls in cdot-rsu-api.js so we don't lose track of where we are calling the backend.

Reply:
This method has been moved to the `CdotApi` class and renamed to 'postContactSupport'.